### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,21 +42,21 @@ lint:
...
 	RETRIES=0 && while ! wget -T 10 -q -O /dev/null http://localhost:6060/v1/namespaces ; do sleep 1 ; echo -n "." ; if [ $${RETRIES} -eq 10 ] ; then echo " Timeout, aborting." ; exit 1 ; fi ; RETRIES=$$(($${RETRIES}+1)) ; done
-	cat ./tmp/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./tmp/clair/%.json" -l ./tmp/clair/clair.log %'; \
+	cat ./docker-image/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./clair/%.json" -l ./clair/clair.log %'; \
 	XARGS_EXIT=$$?; \
-	if [ $${XARGS_EXIT} -eq 123 ]; then find ./tmp/clair/wyrihaximusnet -type f | sed 's/^/-Fjson=@/' | xargs -d'\n' curl -X POST ${WALLE_REPORT_URL} -F channel=team_oz -F buildUrl=https://circleci.com/gh/wyrihaximusnet/docker-php/${CIRCLE_BUILD_NUM}#artifacts/containers/0; else exit $${XARGS_EXIT}; fi
\ No newline at end of file
+	if [ $${XARGS_EXIT} -eq 123 ]; then find ./clair/wyrihaximusnet -type f | sed 's/^/-Fjson=@/' | xargs -d'\n' curl -X POST ${WALLE_REPORT_URL} -F channel=team_oz -F buildUrl=https://circleci.com/gh/wyrihaximusnet/docker-php/${CIRCLE_BUILD_NUM}#artifacts/containers/0; else exit $${XARGS_EXIT}; fi
```
Comment (bendavies) [inline Makefile]: reference to circleci here
unfortunately there is no current way to get the github actions build number

Comment (WyriHaximus) [reply Makefile]: There is, you'd need `jq` but you can het it from the event file. Let me fix that and tag you in the PR :)

Comment (bendavies) [reply Makefile]: hmm that must be new! thanks

Comment (bendavies) [reply Makefile]: any luck? still not available AFAICS
https://github.community/t5/GitHub-Actions/Get-Runs-Url/m-p/38506

Comment (WyriHaximus) [reply Makefile]: Not luck yet, been trying to get a json string of the github variable in the runner but no luck so far

**Pull request merged**